src/src/rx-serial/SerialSUMD.h

```diff
@@ -0,0 +1,67 @@
+#include "SerialIO.h"
+
+static const uint16_t crc16_table[256] = {
```

Comment:
Instead of adding the table, is the poly known?

Reply:
The CRC calculation is shown in this document https://www.deviationtx.com/media/kunena/attachments/98/HoTT-SUMD-Spec-REV01-12062012-pdf.pdf on page 6.
It has the polynomial listed there, so the CRC table is not required.

Reply:
Yes, I know the spec. I mean it's a tradeoff between speed and flash usage. If you want to save flash I'm happy to put in the polynomial calc. Will update this PR.